### DISCUSSION TITLE
Login Not Working on Server

### DISCUSSION THREAD

**Discussion by burhanahmed92** [filamentphp/filament#2798] [Help]
I deployed my project on an ubuntu server. There is an issue with auth. When I enter my admin user credential then page reload itself. Gives no error.
I also tried refresh migration and creating a new admin user but it's not working.
Any suggestion?

**Comment by danharrin** [comment]
Any console errors? Sounds like your server is not configured properly to support Livewire

**Comment by burhanahmed92** [comment]
it shows this response when i press login. Seems livewire is not getting values from input fields

![image](https://user-images.githubusercontent.com/1272261/174244816-6f954ae9-7d88-447c-b32a-c7eece53a8bd.png)

**Reply by danharrin** [reply]
Is livewire.js a 404?
